legend-engine-xts-java/legend-engine-xt-javaPlatformBinding-pure/src/main/resources/core_java_platform_binding/legendJavaPlatformBinding/planConventions/stringLibrary.pure

```diff
@@ -87,8 +87,9 @@ function meta::pure::executionPlan::platformBinding::legendJava::library::string
          fc2(decodeUrl_String_1__String_1__String_1_,                            {ctx,str,charset            | $library->j_invoke('decodeUrl', [$str, $charset], javaString())}),         
          fc2(encodeUrl_String_1__String_1__String_1_,                            {ctx,str,charset            | $library->j_invoke('encodeUrl', [$str, $charset], javaString())}),
 
-         fc2(hash_String_1__HashType_1__String_1_,                               {ctx,text,hashType          | $library->j_invoke('hash', [$text, $hashType], javaString())})
-
+         fc2(hash_String_1__HashType_1__String_1_,                               {ctx,text,hashType          | $library->j_invoke('hash', [$text, $hashType], javaString())}),
+         fc2(jaroWinklerSimilarity_String_1__String_1__Float_1_,                 {ctx,str1,str2              | $library->j_invoke('jaroWinklerSimilarity',[$str1, $str2], javaFloat())}),
```

Comment:
Isn't return type javaDouble? As the Library method is this - 

public static double jaroWinklerSimilarity(String str1, String str2)

Reply:
Will do